src/api/app/assets/javascripts/webui/application.js

```diff
@@ -207,7 +207,7 @@ function change_role(obj) {
         complete: function () {
             $('#' + type + '_spinner').hide();
             $('#' + type + '_table input').animate({opacity: 1}, 200);
-            $('#' + type + '_table input').removeAttr('disabled');
+            $('#' + type + '_table input').prop('disabled', false);
```

Comment:
@coolo :  I switched the `.removeAttr(prop)` to `.prop(prop, false)` per [jQuery's recommendation](https://jquery.com/upgrade-guide/3.0/#attributes). Can you give an example of this failing?